bin/quality

```diff
@@ -8,3 +8,5 @@ bin/rails_best_practices --silent --spec --features -x lib/templates, config
 
 bin/bundle-audit update
 bin/bundle-audit
+
+scss-lint app/assets/stylesheets/
```

Comment:
@rustemginiyatullin to use 

1. let's move `scss-lint` right after `rails_best_practices`
2. you need binstubs and commit `scss-lint` to use it from console `bundle binstubs scss-lint`
3. as I can see it has some sort of configuration files, so let's have in stored inside `config` directory, and use it inside `ci` script

```bash
#!/usr/bin/env sh

set -e

bin/rubocop
bin/brakeman --quiet --skip-libs --exit-on-warn
bin/rails_best_practices --silent --spec --features -x lib/templates, config
bin/scss-lint --config config/scss-lint.yml

bin/bundle-audit update
bin/bundle-audit
```